apps/artist/routes.coffee

```diff
@@ -20,13 +21,18 @@ query = """
 module.exports.index = (req, res, next) ->
   artist = new Artist id: req.params.id
   artist.fetch
-    cache: true
-    success: ->
-      artist.maybeFetchAndSetFeaturedBio =>
-        res.locals.sd.ARTIST = artist.toJSON()
-        showAuctionLink = artist.get('display_auction_link')
-        res.render 'page', artist: artist, sort: req.query?.sort, showAuctionLink: showAuctionLink
-    error: res.backboneError
+  Q.all([
+    artist.fetch(cache: true)
+    artist.maybeFetchAndSetFeaturedBio()
+    artist.fetchArtworks(data: {size: 50})
+  ]).then ([artistData, partnerArtists, artworks]) ->
+    res.locals.sd.ARTIST = artistData
+    res.locals.sd.ARTWORKS = artworks
```

Comment:
So that we can [send](https://github.com/artsy/microgravity/pull/93/files#diff-15c581b95b1b523b1ab4c35946c5a9a2R106) artwork ids on `/artist/<slug>` pages.